src/main/java/com/alkemy/ong/controller/ActivityController.java

```diff
@@ -1,4 +1,27 @@
 package com.alkemy.ong.controller;
 
+import com.alkemy.ong.dto.ActivityDto;
+import com.alkemy.ong.service.ActivityService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/activities")
 public class ActivityController {
+
+    private final ActivityService activityService;
+
+    @PostMapping("/create")
```

Comment:
without the "/ create" route I think it looks nicer

Reply:
I think the same, thanks for your observation.